synapse/lib/types.py

```diff
@@ -1408,6 +1408,27 @@ def postTypeInit(self):
         self.setNormFunc(list, self._normPyTuple)
         self.setNormFunc(tuple, self._normPyTuple)
 
+        self.formfilter = None
+        if (filt := self.opts.get('formfilter')) is not None:
```

Comment:
Could we just break this out into a `forms` key and an `interfaces` key to keep the type opts flat?

Reply:
Yeah, I went back and forth on how it should be structured, I can switch them to being separate.